javascript/statements.json

```diff
@@ -1237,9 +1237,98 @@
             }
           }
         },
+        "import_attributes": {
+          "__compat": {
+            "description": "Import attributes",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "impl_url": "https://crbug.com/v8/13856"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": "1.37"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1736059"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
```

Comment:
I tried linking https://github.com/nodejs/node/pull/50140, but it looks like the JSON schema does not accept GitHub URLs.